src/main/java/ui/Ui.java

```diff
@@ -1,29 +1,28 @@
 package ui;
 
 import dukeoutput.DukeOutput;
-import parser.Parser;
+import parser.CommandParser;
 import command.Command;
 import constant.DukeConstants;
-import service.DukeService;
+import storage.Storage;
+import tasklist.TaskList;
 
 import java.util.Objects;
 import java.util.Scanner;
 
 public class Ui {
 
-    private final DukeService dukeService = new DukeService();
-    private final DukeOutput dukeOutput = new DukeOutput();
     private final Scanner scanner = new Scanner(System.in);
```

Comment:
Maybe it would be easier to differentiate your variables if you constants had the form SOME_CONSTANT?